src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/OrderItemsController.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Mime;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using NHSD.BuyingCatalogue.Ordering.Api.Models;
+using NHSD.BuyingCatalogue.Ordering.Common.Constants;
+
+namespace NHSD.BuyingCatalogue.Ordering.Api.Controllers
+{
+    [Route("api/v1/orders/{orderId}")]
+    [ApiController]
+    [Produces(MediaTypeNames.Application.Json)]
+    [Authorize(Policy =  PolicyName.CanAccessOrders)]
+    public sealed class OrderItemsController : ControllerBase
+    {
+        [HttpGet]
+        [Route("order-items")]
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Performance", "CA1822:Mark members as static", Justification = "Swagger does not allow static files")]
+        public ActionResult<List<string>> Get(string orderId)
+        {
+            if(orderId is null)
```

Comment:
It is just canned data but this will obviously need to be changed to return NotFound, rather than thow an exception.

Reply:
👍